src/view.cpp

```diff
@@ -117,10 +117,23 @@ void View::show_error(const std::string& msg)
 	set_status(msg);
 }
 
+bool View::run_commands(const std::vector<MacroCmd>& commands)
+{
+	for (auto command : commands) {
```

Comment:
```suggestion
	for (auto command : commands) {
	    if (formaction_stack_size() == 0) {
	        return true;
	    }
```

Without this check, this PR can be provoked to segfault using e..g. `run-at-startup quit; quit`.

Reply:
Done and squashed into existing commit: https://github.com/newsboat/newsboat/pull/1134/commits/451f1b5e00d5623334260c08e2aa24aa52b53feb#diff-64be1adcbd17050bc44ab8b1eea4771aR218

Reply:
I remember thinking about this but apparently forgot about it.
Thanks for pointing this out!

Fixed in https://github.com/newsboat/newsboat/pull/1134/commits/1730dc1091034ec82af07b3a2ccc4f36fd9ecf87.